src/EnergyPlus/Boilers.hh

```diff
@@ -65,169 +66,175 @@ namespace Boilers {
+            void setDesignVolumeFlowRate(Real64 const flowRate);
+
+            // set the sizing factor for the boiler, if sizingFactor <= 0.0, it is set to 1.0
+            void setDesignSizingFactor(Real64 const sizingFactor);
+            void setLoopNumber(int const loopNumber);
+            void setDesignOutletTemperature(Real64 const temperature);
+
+            // set the outlet limit temperature, if temperature <= 0.0, set it to 99.9
+            void setDesignOutletTemperatureLimit(Real64 const temperature);
+
+            // Default Constructor
+            BoilerObject()
+                : FuelType(0), TypeNum(0), LoopNum(0), LoopSideNum(0), BranchNum(0), CompNum(0), designNominalCapacity_(0.0),
+                  designNominalCapacityWasAutoSized_(false), designEfficiency_(0.0), designOutletTemperature_(0.0),
+                  designFlowMode_(FlowModeType::Default), ModulatedFlowSetToLoop(false), ModulatedFlowErrDone(false), designVolumeFlowRate_(0.0),
+                  designVolumeFlowRateWasAutoSized_(false), designMassFlowRate_(0.0), designSizingFactor_(0.0), nodeHotWaterInletIndex_(0),
+                  nodeHotWaterOutletIndex_(0), designMinPartLoadRatio_(0.0), designMaxPartLoadRatio_(0.0), designOptimalPartLoadRatio_(0.0),
+                  efficiencyCurveTemperatureMode_(TemperatureEvaluationModeType::Default), curveEfficiencyIndex_(0),
+                  efficiencyCurveType_(EfficiencyCurveType::Default), designOutletTemperatureLimit_(0.0), designParasiticElectricalLoad_(0.0),
+                  EffCurveOutputError(0), EffCurveOutputIndex(0), CalculatedEffError(0), CalculatedEffIndex(0), FaultyBoilerFoulingFlag(false),
+                  FaultyBoilerFoulingIndex(0), FaultyBoilerFoulingFactor(1.0), doOneTimeInitialisation_(true), doEnvironmentInitialisation_(true),
+                  operatingLoad_(0.0), operatingHeatingEnergy_(0.0), operatingParasiticElectricalPower_(0.0),
+                  operatingParasiticElectricalConsumption_(0.0), operatingMassFlowRate_(0.0), operatingInletTemperature_(0.0),
+                  operatingPartLoadRatio_(0.0), operatingFuelUseRate_(0.0), operatingFuelUse_(0.0)
+            {
+            }
+
+        private:
+            int FuelType;                                                       // resource type assignment
+            int TypeNum;                                                        // plant loop type identifier
+            int LoopNum;                                                        // plant loop connection
+            int LoopSideNum;                                                    // plant loop side connection
+            int BranchNum;                                                      // plant loop branch connection
+            int CompNum;                                                        // plant loop component connection
+            Real64 designNominalCapacity_;                                      // W - design nominal capacity of Boiler
```

Comment:
Please use a `m_` prefix for private variables, not a trailing underscore.

Reply:
No problem to modify the private variable naming style. There seemed to be some discussion on this topic [here](https://github.com/NREL/EnergyPlus/wiki/Coding-Standards-Part-1-Style#distinguish-private-object-data) so I was a bit uncertain. Currently it matches the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Variable_Names).

@Myoldmopar @jasondegraw Would it be helpful if I submit a pull request to update the style guide based on the comments received in reviewing this pull request?